backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -369,6 +375,8 @@ def test_api__user_live_messages_endpoint_with_GRIP_proxy__ok__content_tag__asyn
             assert update_user_request.status_code == 200
             tag_event = next(client_events)
             content_tag_event = next(client_events)
+        if sqlalchemy_database == "mariadb":
+            request.node.add_marker(pytest.mark.xfail(reason="unclear"))
```

Comment:
For unclear reason, this test may fail sometime with mariadb. It's complicated to reproduce.
What's seems happening is that the ordering of message received is sometime wrong. It's unclear if the issue happened only in mariadb because of the db itself or if the issue is widely present but doesn't occur on other database because of some "luck".